ui/admin/app/templates/scopes/scope.hbs

```diff
@@ -33,218 +33,314 @@
   />
 {{/if}}
 
-<Rose::Layout::Sidebar as |sidebarLayout|>
-
-  {{#if this.session.isAuthenticated}}
-    <sidebarLayout.sidebar>
-
-      {{#if @model.isGlobal}}
-        <Rose::Nav::Sidebar @title={{t 'titles.general'}} as |nav|>
-          <nav.link @route='scopes.scope.scopes'>
-            <Rose::Icon @name='flight-icons/svg/org-16' @size='medium' />
-            {{t 'resources.org.title_plural'}}
-          </nav.link>
-
-          {{#if (can 'navigate scope' @model collection='session-recordings')}}
-            <nav.link @route='scopes.scope.session-recordings'>
-              <Rose::Icon @name='flight-icons/svg/film-16' @size='medium' />
-              {{t 'resources.session-recording.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if (can 'navigate scope' @model collection='storage-buckets')}}
-            <nav.link @route='scopes.scope.storage-buckets'>
-              <Rose::Icon
-                @name='flight-icons/svg/cloud-upload-16'
-                @size='medium'
-              />
-              {{t 'resources.storage-bucket.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if (can 'navigate scope' @model collection='policies')}}
-            <nav.link @route='scopes.scope.policies'>
-              <Rose::Icon
-                @name='flight-icons/svg/hourglass-16'
-                @size='medium'
-              />
-              {{t 'resources.policy.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if (can 'navigate scope' @model collection='aliases')}}
-            <nav.link @route='scopes.scope.aliases'>
-              <Rose::Icon @name='flight-icons/svg/at-sign-16' @size='medium' />
-              {{t 'resources.alias.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if
-            (and
-              (can 'navigate worker' @model collection='workers')
-              (feature-flag 'byow')
-            )
-          }}
-            <nav.link @route='scopes.scope.workers'>
-              <Rose::Icon @name='flight-icons/svg/cpu-16' @size='medium' />
-              {{t 'titles.workers'}}
-            </nav.link>
-          {{/if}}
-        </Rose::Nav::Sidebar>
+{{#if this.session.isAuthenticated}}
```

Comment:
Is this check being done twice since it looks like there's one here too?
https://github.com/hashicorp/boundary-ui/blob/9de44d5f3a6bfb7383f602b1e2f7d77c015a1986/ui/admin/app/templates/application.hbs#L9

Reply:
ah good catch